src/DDTrace/Integrations/Yii/YiiIntegration.php

```diff
@@ -151,4 +151,23 @@ function (SpanData $span, $args) use ($service) {
             }
         );
     }
+
+    /**
+     * Returns the resource name if it can be detected from the invocation args, null otherwise.
+     *
+     * @param array invocation params of teh Module and Controller::runAction method.
+     * @return string|null the obtained resource name or null if it cannot be detected.
+     */
+    public static function extractResourceNameFromRunAction($args)
+    {
+
+        if (isset($args[0]) && is_string($args[0])) {
```

Comment:
Minor suggestion here to use a fully qualified funtion name.
```suggestion
        if (isset($args[0]) && \is_string($args[0])) {
```